.github/workflows/find-inactive-collaborators.yml

```diff
@@ -0,0 +1,34 @@
+name: Find inactive collaborators
+
+on:
+  schedule:
+    # Run every Monday at 4:05 AM UTC.
+    - cron: 5 4 * * 1
+
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: lts/*
```

Comment:
Mostly just curious, is this just to try make the script run in a more deterministic context than whatever GH's defaults are? This is a bit new to me, does not seem to be referenced elsewhere, etc...